dfastmi/gui/dialog_view.py

```diff
@@ -390,81 +512,76 @@ def activate_dialog(self) -> None:
+        fil = QFileDialog.getOpenFileName(
+            caption=self._view_model.gui_text("select_cfg_file"), filter="Config Files (*.cfg)"
         )
         filename = fil[0]
         if filename != "":
-            if not self.view_model.load_configuration(filename):
-                self.showError(self.view_model.gui_text("file_not_found", prefix="", dict={"name": filename}))
+            if not self._view_model.load_configuration(filename):
+                self._showError(self._view_model.gui_text("file_not_found", prefix="", dict={"name": filename}))
 
-    def menu_save_configuration(self) -> None:
+    def _menu_save_configuration(self) -> None:
         """
         Ask for a configuration file name and save GUI selection to that file.
 
         Arguments
         ---------
         None
         """
-        fil = QtWidgets.QFileDialog.getSaveFileName(
-            caption=self.view_model.gui_text("save_cfg_as"), filter="Config Files (*.cfg)"
+        fil = QFileDialog.getSaveFileName(
+            caption=self._view_model.gui_text("save_cfg_as"), filter="Config Files (*.cfg)"
         )
         filename = fil[0]
         if filename != "":
-            self.view_model.save_configuration(filename)
-
+            self._view_model.save_configuration(filename)
 
-    def menu_about_self(self) -> None:
+    def _menu_about_self(self) -> None:
         """
         Show the about dialog for D-FAST Morphological Impact.
 
         Arguments
         ---------
         None
         """
-        msg = QtWidgets.QMessageBox()
-        msg.setIcon(QtWidgets.QMessageBox.Information)
+        msg = QMessageBox()
+        msg.setIcon(QMessageBox.Information)
         msg.setText("D-FAST Morphological Impact " + dfastmi.__version__)
         msg.setInformativeText("Copyright (c) 2024 Deltares.")
-        msg.setDetailedText(self.view_model.gui_text("license"))
-        msg.setWindowTitle(self.view_model.gui_text("about"))
-        msg.setStandardButtons(QtWidgets.QMessageBox.Ok)
+        msg.setDetailedText(self._view_model.gui_text("license"))
+        msg.setWindowTitle(self._view_model.gui_text("about"))
+        msg.setStandardButtons(QMessageBox.Ok)
         msg.exec_()
 
-
-    def menu_about_qt(self) -> None:
+    def _menu_about_qt(self) -> None:
         """
         Show the about dialog for Qt.
 
         Arguments
         ---------
         None
         """
-        QtWidgets.QApplication.aboutQt()
-
+        QApplication.aboutQt()
 
-    def menu_open_manual(self):
+    def _menu_open_manual(self):
         """
         Open the user manual.
 
         Arguments
         ---------
         None
         """        
-        subprocess.Popen(self.view_model.manual_filename, shell=True)
+        subprocess.Popen(self._view_model.manual_filename, shell=True)
     
-    def openFileLayout(self, win, myWidget, key: str):
+    def _openFileLayout(self, win, myWidget, key: str, enabled: bool):
```

Comment:
This feels the same as _openFolderLayout

Reply:
yes they do! will look into this

Reply:
can't get it similar without a boolean switch... do you have thoughs? or ideas? let look into this later

Reply:
[DFAST-187](https://issuetracker.deltares.nl/browse/DFAST-187)